src/core/tobot_application.cpp

```diff
@@ -2,35 +2,20 @@
 
 #include "utilities/logger.h"
 #include "project_config.h"
+#include "edl/edl_initializer.h"
 
 using namespace Tobot::Core;
 
 void TobotApplication::initialize()
 {
     LOG_INFO("%s version %s.%s.%s", PROJECT_NAME, PROJECT_VERSION_MAJOR, PROJECT_VERSION_MINOR, PROJECT_VERSION_PATCH);
-    if (SDL_Init(SDL_INIT_EVERYTHING)) {
-        LOG_CRITICAL("SDL could not initialize! SDL_Error: %s\n", SDL_GetError());    
+    // Initialize SDL subsystems
+    if (Tobot::EDL::EDLInitializerInitialize(Tobot::EDL::SDL_CORE_INIT_EVERYTHING | 
+                                                Tobot::EDL::SDL_IMAGE_INIT_PNG | 
+                                                Tobot::EDL::SDL_TTF_INIT | 
+                                                Tobot::EDL::SDL_MIXER_INIT_MP3))
```

Comment:
Method body here or not?

Reply:
I don't think I understand the question / remark. The method body is in edl_initializer.cpp and we exit if the subsystems were not initialized properly. But looks pretty messy without curly braces. This could be adapted.